pdfminer/pdffont.py

```diff
@@ -76,7 +78,33 @@ def get_widths2(seq):
 #assert get_widths2([1,2,3,4,5]) == {1:(3, (4,5)), 2:(3, (4,5))}
 #assert get_widths2([1,[2,3,4,5],6,[7,8,9]]) == {1:(2, (3,4)), 6:(7, (8,9))}
 
-
+def computeStat(it):
+	stat=defaultdict(lambda: defaultdict(int))
+	for fd in it:
+		for cat, size in itemz(fd):
```

Comment:
These variable names (`cat` and `size`) confuse me. They are the key-value pairs for each font metric for a specific font, right? 

Maybe call them `font_descriptor_name` and `font_descriptor_name`?